docs/core/configuration.md

```diff
@@ -605,13 +605,13 @@ Here's a brief summary of the timeouts:
   on the new height (this gives us a chance to receive some more precommits,
   even though we already have +2/3)
 
-### The effect of `timeout_propose` on the proposer selection process
+### The adverse effect of using inconsistent `timeout_commit` in a network
 
-Here's an interesting question. What if the particular validator sets a very
-small `timeout_propose`?
+Here's an interesting question. What happens if a particular validator sets a
+very small `timeout_commit`, as compared to the rest of the network?
 
 Imagine there are only two validators in your network: Alice and Bob. Bob sets
-`timeout_propose` to 1s. Alice uses the default value of 3s. Bob will create
+`timeout_commit` to 1s. Alice uses the default value of 3s. Bob will create
 blocks ~ every second, Alice - every 3 seconds (given `create_empty_blocks` is
 `true`). Let's say they both have an equal voting power. Given the proposer
```

Comment:
The default for `create_empty_blocks` is `true`, maybe it is the case to mention this, something like "given the default value of..." or "given that that ... was not changed to `false`.